src/Sulu/Component/Content/Compat/PropertyParameter.php

```diff
@@ -15,7 +15,7 @@
 use JMS\Serializer\Annotation\Type;
 
 /**
- * Represents a parameter of a property.
+ * @internal this class can be removed at any time and you should not build on top of it.
  */
 class PropertyParameter implements \JsonSerializable
```

Comment:
Same as for the Metadata class need to have a deeper look at the classes using this.

Reply:
few other classes also readded mostly exception which we need also move to admin bundle